docs/Changelog.md

```diff
@@ -27,6 +27,10 @@
 * [#5033](https://github.com/TouK/nussknacker/pull/5033) Updated Scala 2.13 to 2.13.12
 * [#4887](https://github.com/TouK/nussknacker/pull/4887) New parameter validator - `ValidationExpressionParameterValidator` that allows to use SpEL (or any other) expression to validate parameters
 * [#5077](https://github.com/TouK/nussknacker/pull/5077) Add an option to set schema on connections in SQL enricher
+* [#4953](https://github.com/TouK/nussknacker/pull/4953) Add stricter validation in base components:
```

Comment:
@coutoPL @pielas This changelog is ok for previous and further Realease Managers? Does it match "Added entry in Changelog.md describing the change from the perspective of a public distribution user" rule?

Reply:
Forgot to push that change - I propose a generic "Improved node validation" here in changelog and this extended description 

"Add stricter node validation:
  * Boolean expressions in Switch and Filter nodes now required not null values
  * Variable values in MapVariable, FragmentOutput and Variable are now mandatory
  * Field names in MapVariable, FragmentOutput are required to be unique"

Reply:
LGTM